editor/assets/stylesheets/_z-index.scss

```diff
@@ -13,8 +13,8 @@ $z-layers: (
 	'.editor-post-visibility__dialog': 30,
 	'.editor-post-schedule__dialog': 30,
 	'.editor-block-mover': 30,
-	'.components-notice-list': 100000,
-	'.components-popover': 100000,
```

Comment:
I agree about the notices, I was talking about the popovers (They can be full width and have a grayed background to cover the rest of the page)